lib/tasks/puppet.rake

```diff
@@ -23,140 +23,6 @@ namespace :puppet do
       end
```

Comment:
For a separate PR, but I'd suggest deprecating the above in favor of `/etc/puppetlabs/puppet/node.rb --push-facts` which provides the same functionality and even has a `--push-facts-parallel` option.